pack_test.go

```diff
@@ -37,124 +35,128 @@ func Test_Pack_Default(t *testing.T) {
 	s := memory.New()
 
 	// prepare test content
-	layer_1 := []byte("hello world")
+	blob_1 := []byte("hello world")
 	desc_1 := ocispec.Descriptor{
 		MediaType: "test",
-		Digest:    digest.FromBytes(layer_1),
-		Size:      int64(len(layer_1)),
+		Digest:    digest.FromBytes(blob_1),
```

Comment:
nit: should we use `content.NewDescriptorFromBytes()` to construct the descriptors?

Reply:
Refactored.